src/pylibmc/client.py

```diff
@@ -36,7 +36,7 @@ def _unpack_addr(spec, port=None, weight=1):
     else:
         addr = spec
 
-    if isinstance(port, str) and ":" in port:
+    if isinstance(port, (str, unicode)) and ":" in port:
```

Comment:
This will not work in Python 3.x, and Python 2.x has been removed as a target platform.